src/jump_moi_overloads.jl

```diff
@@ -34,7 +34,7 @@ function _moi_get_result(model::MOIU.CachingOptimizer, args...)
     end
     return MOI.get(model, args...)
 end
-function MOI.get(model::JuMP.Model, attr::ForwardOutVariablePrimal, var_ref::JuMP.VariableRef)
+function MOI.get(model::JuMP.Model, attr::Attr, var_ref::JuMP.VariableRef) where {Attr <: Union{ForwardOutVariablePrimal, BackwardOutConstraint, BackwardOutObjective}}
```

Comment:
For `BackwardOutConstraint`, the third argument should be a constraint reference. For `BackwardOutObjective`, there should be no third argument.